packages/components/src/icon/test/index.js

```diff
@@ -21,7 +21,7 @@ describe( 'Icon', () => {
 	it( 'renders nothing when icon omitted', () => {
 		render( <Icon data-testid={ testId } /> );
 
-		expect( screen.queryByTestId( testId ) ).not.toBeInTheDocument();
+		expect( screen.queryByTestId( testId ) ).toBeNull();
```

Comment:
Do you think this additional fix is reasonable?

Reply:
No, I think `.not.toBeInTheDocument()` is ideally expressing our goal here; `.toBeNull()` makes the assertion vague and unclear, while what we have already is pretty transparent IMHO.

Reply:
`.not.toBeVisible()` won't work by design, an element can't be visible if it doesn't exist in the DOM tree, that's why we check with `.not.toBeInTheDocument()`.

Reply:
Thanks, @t-hamano 🙌